Provider/ProviderProxy.php

```diff
@@ -69,9 +69,11 @@ public function getName()
      */
     public function shorten(LinkInterface $link)
     {
+        $func = array($this->provider, 'shorten');
+
         $event = $this->startProfiling($link->getLongUrl());
 
-        $this->provider->shorten($link);
+        call_user_func_array($func, func_get_args());
```

Comment:
Can you plz update it like this and so remove the previous assignment:
`call_user_func_array(array($this->provider, 'shorten'), func_get_args());`